tests/load/load-test.js

```
@@ -0,0 +1,178 @@
+    eventsConnectDuration.add(res.timings.duration);
+
+    const hasEventStream = res.headers['Content-Type']?.includes('text/event-stream');
+    const hasData = res.body?.includes('data:') || res.status === 200;
+
+    check(res, {
+      'SSE returns 200': (r) => r.status === 200,
+      'SSE has text/event-stream content-type': () => hasEventStream || hasData,
+    });
+
+    if (res.status !== 200) {
+      sseErrorRate.add(1);
```

ESLint

[error] Delete `;`

(prettier/prettier)